doc/api/api_changes/2017-09-22-AL-removal-of-deprecated-features.rst

```diff
@@ -0,0 +1,17 @@
+Removal of deprecated features
+``````````````````````````````
+
+The ``matplotlib.finance``, ``mpl_toolkits.exceltools`` and
```

Comment:
Is the finance toolkit now in another package?

Reply:
done